python/distrdf/dask/check_reducer_merge.py

```diff
@@ -341,6 +341,57 @@ def test_redefine_one_column(self, connection):
         assert sum_before.GetValue() == 10.0
         assert sum_after.GetValue() == 20.0
 
+    def test_distributed_stddev(self, connection):
+        """Test support for the StdDev action."""
+        df = (
+            Dask.RDataFrame(10_000, daskclient=connection)
+                .Define("v", "gRandom->Gaus()")
```

Comment:
here and below it would be more robust to use a fixed series of numbers rather than sampling them